src/main/scala/wasm/MiniWasm.scala

```diff
@@ -437,7 +464,8 @@ case class Evaluator(module: ModuleInstance) {
         })
     }
     if (instrs.isEmpty) println("Warning: nothing is executed")
-    eval(instrs, List(), Frame(ArrayBuffer(I32V(0))), halt, List(halt))
+
+    eval(instrs, List(), Frame(ArrayBuffer(I32V(0),I32V(0))), halt, List(halt))
```

Comment:
Meeting notes: frame's length should depends on the number of local variables, which declared in `main`'s fields.